app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,56 @@
+class CommentsController < ApplicationController
+  def new
+    bathroom = Bathroom.find(params[:bathroom_id])
+    @comment = bathroom.comments.new
+  end
+
+  def create
+    if logged_in?
+      bathroom = Bathroom.find(params[:bathroom_id])
+      @comment = bathroom.comments.new(comment_params)
+      @comment.user_id = session[:user_id]
```

Comment:
Remember you can merge the user into the params and not have to keep repeating this